javascript-methods/main.js

```diff
@@ -0,0 +1,67 @@
+var one = 30;
+var two = 40;
+var three = 50;
+
+var maximumValue = Math.max(one,two,three);
+
+console.log('Value of maximumValue is: ',maximumValue);
+
+var heroes = ['superman','batman','spiderman','groot'];
+
+var randomNumber = Math.random();
+
+var randomNumber = randomNumber * heroes.length;
+
+var randomIndex = Math.floor(randomNumber);
+
+console.log('Value of random index: ', randomIndex);
+
+var randomHero = heroes[randomIndex];
+
+console.log('Value of randomHero: ',randomHero);
+
+
+var library = [
+  book1 = {
```

Comment:
you should not have undeclared variables like this here. when you want to add objects into an array I would recommend the following: 

```javascript
var arrayWithObjects = [{},{},{}]
```